test/unit/config/configFileTest.ts

```diff
@@ -195,7 +195,7 @@ describe('Config', () => {
         config.unlinkSync();
         assert(false, 'should throw');
       } catch (e) {
-        expect(e.name).to.equal('TargetFileNotFound');
```

Comment:
you can also use chai assert to assert that it is the error of the expected type.  It's bonus a useful test assertion *and* narrows TS so all the props are valid

`assert(e instanceOf Error)` (or could be SfError if you know it should be that)